### PR TITLE
[FEATURE}: Added dictionary widget

### PR DESCRIPTION
**Pull request by Ananya-Bhardwaj**
Fixes the issue #56 

Dictionary widget fetches the definition, pronunciation, synonyms and antonyms of the word entered by the user

# Result 

image

### CODE REVIEW

**Comment by mohitahlawat2001** [comment]
getting this error
Failed to compile.
Module not found: Error: Can't resolve 'react-icons/ai' in '/vercel/path0/src/widgets/DictionaryWidget'
Error: Command "npm run build" exited with 1

File: src/widgets/DictionaryWidget/DictionaryContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { AiFillSound } from 'react-icons/ai';
```
Comment (mohitahlawat2001) [inline src/widgets/DictionaryWidget/DictionaryContainer.jsx:2]: error in this line

**Review by mohitahlawat2001 — CHANGES_REQUESTED** [review]

File: src/widgets/DictionaryWidget/DictionaryContainer.jsx (hunk 1 above)
Comment (Ananya-Bhardwaj) [reply src/widgets/DictionaryWidget/DictionaryContainer.jsx:2]: Okay let me check why is it coming?

Comment (mohitahlawat2001) [reply src/widgets/DictionaryWidget/DictionaryContainer.jsx:2]: you need to upload package.json as it does not contain this package react-icon

Comment (Ananya-Bhardwaj) [reply src/widgets/DictionaryWidget/DictionaryContainer.jsx:2]: Okay, I'll do that

**Pull request merged**